crowbar_framework/spec/models/backup_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "spec_helper"
+
+describe Backup do
+  describe "Backup creation" do
+    let(:backup) { Backup.create("testbackup") }
+
+    context "new backup" do
+      it "checks the type" do
+        expect(backup).to be_an_instance_of(Backup)
+      end
+
+      it "has attributes" do
+        [:name, :created_at, :filename, :path].each do |attribute|
+          expect(backup).to respond_to(attribute)
+        end
+      end
+
+      context "validation" do
+        it "is valid" do
+          expect(backup.save).to be true
+        end
+      end
+    end
+  end
+
+  describe "Backup object" do
+    let(:created_at) { Time.now.strftime("%Y%m%d-%H%M%S") }
```

Comment:
Rails/TimeZone: Do not use `Time.now.strftime` without zone. Use one of `Time.zone.now.strftime`, `Time.now.strftime.current`, `Time.now.strftime.in_time_zone`, `Time.now.strftime.utc`, `Time.now.strftime.getlocal`, `Time.now.strftime.iso8601`, `Time.now.strftime.jisx0301`, `Time.now.strftime.rfc3339`, `Time.now.strftime.to_i`, `Time.now.strftime.to_f` instead. (https://github.com/bbatsov/rails-style-guide#time, http://danilenko.org/2012/7/6/rails_timezones)